### PR TITLE
Add more integration tests

### PR DESCRIPTION
**Pull request by noituri**
Snapshots: https://github.com/membraneframework-labs/video_compositor_snapshot_tests/pull/29

### CODE REVIEW

File: compositor_integration_tests/src/video_decoder.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl VideoDecoder {
     }
 
     pub fn decode(&mut self, packet: rtp::packet::Packet) -> Result<()> {
-        let pts = packet.header.timestamp as f64 / 90000.0;
+        let pts = packet.header.timestamp as f64 / 90000.0 * 1_000_000.0;
```
Comment (noituri) [inline compositor_integration_tests/src/video_decoder.rs:50]: Somehow it got reverted in previous PR

File: compositor_integration_tests/src/common.rs [hunk 2]
```diff
@@ -38,3 +40,30 @@ pub fn output_dump_from_disk<P: AsRef<Path>>(path: P) -> Result<Bytes> {
...
+
+        let packet =
+            rtp::packet::Packet::unmarshal(&mut dump.slice(read_bytes..(read_bytes + packet_len)))?;
+        read_bytes += packet_len;
+
+        let packet_pts = match packet.header.payload_type {
+            96 => Duration::from_secs_f64(packet.header.timestamp as f64 / 90000.0),
+            97 => Duration::from_secs_f64(packet.header.timestamp as f64 / 48000.0),
+            _ => continue,
+        };
```
Comment (WojciechBarczynski) [inline compositor_integration_tests/src/common.rs:60]: I think we want to return an error when LC sends RTP packet with a payload type different than 96 or 97

File: compositor_integration_tests/src/tests/unregistering.rs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        "image_id": "image_1",
+    }))?;
+
+    let new_output_dump = output_receiver.wait_for_output()?;
+    let output_dump_from_disk = output_dump_from_disk("unregistering_test_output.rtp")?;
+
+    compare_video_dumps(
+        &output_dump_from_disk,
+        &new_output_dump,
+        &[Duration::from_millis(1000), Duration::from_millis(3000)],
```
Comment (WojciechBarczynski) [inline compositor_integration_tests/src/tests/unregistering.rs:72]:
```suggestion
        &[Duration::from_secs(1), Duration::from_secs(3)],
```

**Review by WojciechBarczynski — APPROVED** [review]

File: compositor_integration_tests/src/tests.rs [hunk 4]
```diff
@@ -1 +1,14 @@
-mod simple;
+mod audio_mixing;
+mod muxed_video_audio;
+mod required_inputs;
+mod schedule_update;
+mod unregistering;
+
+#[test]
+fn integretion_tests() {
+    required_inputs::required_inputs().unwrap();
```
Comment (wkozyra95) [inline compositor_integration_tests/src/tests.rs:9]: add some TODO comment that it is temporary

**Review by wkozyra95 — APPROVED** [review]

**Pull request merged**